### ISSUE TITLE
Lidarr connection not possible (while Sonarr and Radarr work fine)

### ISSUE THREAD

**Issue by Tom-Furrer** [trossr32/sonarr-radarr-lidarr-autosearch-browser-extension#165]
While Sonarr (develop-branch) and Radarr (develop-branch) work perfectly, my Lidarr (develop-branch) instance seems not to connect to your extension in Google Chrome:

![image](https://github.com/trossr32/sonarr-radarr-lidarr-autosearch-browser-extension/assets/153307907/06bd4945-db2c-46aa-b5dd-ca88a52f83ca)

I have also tried your [suggestion](https://github.com/trossr32/sonarr-radarr-lidarr-autosearch-browser-extension/issues/71#issuecomment-997219542):
http://DS1817Plus:8686/api/system/status?apikey=removed
and
http://ds1817plus:8686/api/v3/system/status?apikey=removed
but I only get a "404 not found" response in my browser.

Calling http://ds1817plus:8686/ works fine in my Google Chrome browser and Lidarr is fully working.

What could I do to configure your extension and let it connect to Lidarr in my docker container?

**Comment by Tom-Furrer** [comment]
By accident I found that
http://ds1817plus:8686/api/v1/system/status?apikey=
would return a JSON result like this

```
{
  "appName": "Lidarr",
  "instanceName": "Lidarr",
  "version": "2.0.7.3849",
  "buildTime": "2023-12-10T19:57:42Z",
  "isDebug": false,
  "isProduction": true,
  "isAdmin": false,
  "isUserInteractive": true,
  "startupPath": "/app/lidarr/bin",
  "appData": "/config",
  "osName": "alpine",
  "osVersion": "3.18.5",
  "isNetCore": true,
  "isLinux": true,
  "isOsx": false,
  "isWindows": false,
  "isDocker": true,
  "mode": "console",
  "branch": "develop",
  "databaseType": "sqLite",
  "databaseVersion": "3.41.2",
  "authentication": "forms",
  "migrationVersion": 74,
  "urlBase": "",
  "runtimeVersion": "6.0.25",
  "runtimeName": "netcore",
  "startTime": "2023-12-11T18:30:16Z",
  "packageVersion": "2.0.7.3849-ls125",
  "packageAuthor": "[linuxserver.io](https://www.linuxserver.io/)",
  "packageUpdateMechanism": "docker"
}
```

How do I have to adjust the configuration?

**Comment by leodoi3** [comment]
Having the same issue

**Comment by trossr32** [comment]
Will be fixed in v2.8.4
